torch/csrc/jit/passes/remove_inplace_ops.cpp

```diff
@@ -112,6 +112,16 @@ void ImplicitCastForBinaryInplaceOps(Block* b) {
       if (originalInputs.at(0) == originalInputs.at(1)) {
         continue;
       }
+
+      auto shape_node = originalInputs.at(0)->node();
+      if ((shape_node->kind() == prim::NumToTensor) &&
+          (shape_node->inputs().at(0)->node()->kind() == aten::size)) {
+        std::cerr
+            << "In tracing mode, shape values obtained from tensor.shape might cause a mismatch "
```

Comment:
I'd replace this with a very short text and the URL of the doc you added.

Maybe:
"In-place op on output of tensor.shape. See https://pytorch.org/docs/master/onnx.html#..."

Also, any way to indicate the source location (file name and line)?